.gitignore

```diff
@@ -33,3 +33,9 @@ bower.json
 
 # Ignore pow environment settings
 .powenv
+
+# Ignore database.yml socket configuration
+/config/database.yml
```

Comment:
As a convention, you want to have a `database.example.yml` file checked into the repo. So that if someone else starts working on this in the future or run the app locally, they can just copy the example config into the real `database.yml` file.